rllib/agents/slateq/slateq.py

```diff
@@ -113,21 +106,17 @@
     "training_intensity": None,
 
     # === Optimization ===
-    # TODO: Unify torch and tf config settings:
-    # Learning rate for adam optimizer for the user choice model.
-    "lr_choice_model": 1e-3,  # Only relevant for framework=torch.
-    # Learning rate for adam optimizer for the q model.
-    "lr_q_model": 1e-3,  # Only relevant for framework=torch.
-    # TODO: Unify torch and tf config settings:
-    # Learning rate for adam optimizer for the q model.
-    "lr": 0.00025,  # Only relevant for framework=tf|tf2.
+    # Learning rate for RMSprop optimizer for the q-model.
+    "lr": 0.00025,
     # Learning rate schedule.
     # In the format of [[timestep, value], [timestep, value], ...]
     # A schedule should normally start from timestep 0.
-    "lr_schedule": None,  # Only relevant for framework=tf|tf2.
+    "lr_schedule": None,
+    # Learning rate for adam optimizer for the user choice model.
+    "lr_choice_model": 1e-3,  # Only relevant for framework=torch.
```

Comment:
how come this is only relevant for torch? If its not in the TF implementation, should we drop this?